sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -39,7 +39,21 @@ import org.apache.spark.util.Utils
 
 
 /**
- * Functions available for DataFrame operations.
+ * Commonly used functions available for DataFrame operations. Using functions defined here provides
```

Comment:
Maybe I am too much caring about this but I hope we don't have arguments too much if it's common or not ...